test/test_overrides.py

```diff
@@ -804,7 +804,6 @@ def test_gradcheck(self):
         # Tensor-likes.
         self.assertEqual(total_used_attrs, {
             'data',
-            'device',
```

Comment:
Previously, we are missing a return before `checkIfNumericalAnalyticAreClose`, now that we add this early exit, we may not access the device attr.